18F/identity-idp
# Implement AcuantCapture React component

**Why**: The single page document authorization upload flow will be implemented using React. As part of this effort, it will be necessary to integrate Acuant's Web SDK for image capturing.

This pull request proposes a minimal set of changes for a basic integration of Acuant image capture using React components.

As implemented, it's largely composed of:

- A context provider, which is responsible for loading and initializing the web SDK
- The user-facing component which prompts the user to begin the capture
- The capture canvas component, required by the Acuant web SDK

It does not intend to:

- ~Localize texts (see #3909)~ **Edit:** Since #3909 is merged, this is now partly implemented here for strings which exist and are likely to remain unchanged (see 7c29c2fa599ddbf74be8035e289a46c2ceff7e4f).
- Incorporate any design elements beyond basic interactions
- Recreate any behavior of the capture flow submission

Testing steps:

- Navigate to http://localhost:3000/
- Log in to your account
- Navigate to http://localhost:3000/verify
- Click checkbox
- Click Continue
- Click link "Upload from your computer"
- Observe "Loading..." indicator while Acuant SDK is loading
- Observe "Take Photo" button appears when Acuant SDK is ready
- Click "Take Photo"
- In a desktop browser, it's likely you'll be prompted to upload an image. Upload a photo.
- Observe photo preview appears after selected

Screenshot:

![Screenshot](https://user-images.githubusercontent.com/1779930/87569301-e6ed6b00-c694-11ea-9485-3b2c7230cad7.png)
